### PR TITLE
RAD-165 Include tinymce as report editor.

### PR DESCRIPTION
**Pull request by oliverkrakora**

### CODE REVIEW

**Comment by teleivo** [comment]
what is the purpose of the links at the bottom left showing different links depending on the format selected?

tiny

feels distracting to me. can you remove it?

**Comment by oliverkrakora** [comment]
Yes that should not be a problem.

**Comment by teleivo** [comment]
@oliverkrakora can you also hide the editor when the report is read-only

**Comment by oliverkrakora** [comment]
Yes I can do that too.

**Comment by dkayiwa** [comment]
@oliverkrakora were you able to actually do it? :)

**Comment by oliverkrakora** [comment]
I have issues testing it because my openmrs machine is currently not working. I will push my changes in the next few days.

**Comment by oliverkrakora** [comment]
I have tested the editor now and I have made all the requested changes. This is how the editor looks like:
bildschirmfoto 2016-04-24 um 18 01 25

File: omod/src/main/webapp/radiologyReportForm.jsp [hunk 1]
```diff
@@ -49,10 +71,10 @@
 				<td><spring:message code="radiology.radiologyReportDiagnosis" /></td>
 				<td><c:choose>
 						<c:when test="${radiologyReport.reportStatus == 'COMPLETED'}">
-							<form:textarea path="reportBody" disabled="true"></form:textarea>
+							<form:textarea id="textarea" path="reportBody" disabled="true"></form:textarea>
```
Comment (teleivo) [inline omod/src/main/webapp/radiologyReportForm.jsp]: can you please set the id to "reportText", so its easier to differentiate it from the form type textarea

Comment (oliverkrakora) [inline omod/src/main/webapp/radiologyReportForm.jsp]: The id of the textarea is now reportText.

**Comment by teleivo** [comment]
@oliverkrakora please rebase onto current master

**Comment by oliverkrakora** [comment]
DONE!

**Comment by teleivo** [comment]
@oliverkrakora thanks a lot!

**Pull request merged**